### ISSUE TITLE
QueryTpmAttestationReport() does not get the proper PCR digest

### ISSUE THREAD

**Issue by arijit8972** [vmware/govmomi#1977]
I am providing a sample code that I have written, 

```
query := types.QueryTpmAttestationReport{This: host.Reference()}
attestationReport, err := methods.QueryTpmAttestationReport(ctx, vmwareClient.RoundTripper, &query)
tpmPCRValues := tpmAttestationReport.Returnval.TpmPcrValues[0]
fmt.Printf("VMware host attestation report : %v\n", tpmPCRValues)
```

Although I see that it is a SHA 256 digest, I am getting just 3 bytes as part of the digest value

Sample Output,

`{{{} SHA256 [45 57 48] } 0}`

I am expecting a 32 byte digest value in case the digest type is SHA 256. Am I missing something ?

**Comment by arijit8972** [comment]
the struct used is,
```
type HostDigestInfo struct {
	DynamicData

	DigestMethod string `xml:"digestMethod"`
	DigestValue  []byte `xml:"digestValue"`
	ObjectName   string `xml:"objectName,omitempty"`
}
```
On changing DigestValue to []int I can get the entire digest, else xml decoder sets digest value to just the last byte of the digest.

**Comment by dougm** [comment]
I hadn't used `QueryTpmAttestationReport` before, and it doesn't return any data against the hosts I'm using. But, I can reproduce what you're seeing with HostSystem config.certificate field, also a `[]byte`. And likewise changing to `[]int` gets all the data. I'll look into it more.

**Comment by ddhawal** [comment]
> I hadn't used `QueryTpmAttestationReport` before, and it doesn't return any data against the hosts I'm using. But, I can reproduce what you're seeing with HostSystem config.certificate field, also a `[]byte`. And likewise changing to `[]int` gets all the data. I'll look into it more.

Confirm if you have TPM/TXT enabled host.

**Comment by arijit8972** [comment]
Thanks for your reply. The xml unmarshaller does not throw any error for the invalid typecasting but instead does it anyways which causes this issue. I am not sure if that is by design but it definitely needs to be fixed here.

**Comment by arijit8972** [comment]
Any update on this issue ?

**Comment by arijit8972** [comment]
Can you please reopen the issue ? It seems to moving through sprints but it has been closed by the bot.

**Comment by geoff-maddock** [comment]
I'm running into this same issue.

I'm calling the QueryTpmAttestationReport on a host with TXT enabled and find that I only get back bytes enough to represent one PCR value.

If I query the same host using the /MOB (typical method of validation), I get a number of PCR values (over a dozen).

If I change this struct in types.go

```
type HostDigestInfo struct {
	DynamicData
	DigestMethod string `xml:"digestMethod"`
	DigestValue  []byte  `xml:"digestValue"`
	ObjectName   string `xml:"objectName,omitempty"`
}
```
Into:
```
type HostDigestInfo struct {
	DynamicData
	DigestMethod string `xml:"digestMethod"`
	DigestValue  []int  `xml:"digestValue"`
	ObjectName   string `xml:"objectName,omitempty"`
}
```

I get the correct response from the report.  

Anyone have a work-around to use in the meanwhile?

**Comment by arijit8972** [comment]
Right now all I could do was fork out of the latest branch and change the structs in my repo and use it.  Hopefully, this would be resolved soon and we would be able to use the parent repo.

**Comment by VitorEstevam** [comment]
Any news on it?

**Comment by dougm** [comment]
Finally got to a fix for this in #3476 and included a couple of new govc tpm commands. I'd not looked at anything tpm related before, but these commands are useful for testing the fix at least. Here's an output example, suggestions welcome:

Summary:
```console
% govc host.tpm.info
Name           Attestation  Last Verified         TPM version  TXT  Message
10.243.168.72  accepted     2024-06-23T06:00:45Z  2.0          false
10.168.189.90  N/A          N/A                   N/A          N/A
```

PCR values:
```console
% govc host.tpm.report -host $host
PCR 0   SHA256  d35f98d83d5e5fd80f90daaa519ce8dce8506132f7a6006490011f8796f9bc22  
PCR 1   SHA256  3d458cfe55cc03ea1f443f1562beec8df51c75e14a9fcf9a7234a13f198e7969  
PCR 2   SHA256  a239522a19f4b1587f1970fc04b9d1745207c5395702fedabee0476f9cd03739  
PCR 3   SHA256  3d458cfe55cc03ea1f443f1562beec8df51c75e14a9fcf9a7234a13f198e7969  
PCR 4   SHA256  bec6b13fd57579f38673d1c51232e0378fcee495eb3e4b66078dd80abbb8cd27  
PCR 5   SHA256  8a8c617a875ff40e2019b7d333508787f164c618acf659f00630e765d4a7f31d  
PCR 6   SHA256  3d458cfe55cc03ea1f443f1562beec8df51c75e14a9fcf9a7234a13f198e7969  
PCR 7   SHA256  15d499c4326fa9a4c934be2d00e0fa5c1e76dd216a73c8ce8858a59818b28d8d  
PCR 8   SHA256  0000000000000000000000000000000000000000000000000000000000000000  
PCR 9   SHA256  0000000000000000000000000000000000000000000000000000000000000000  
PCR 10  SHA256  3dcb2fd152d9e2518a7ccd7f307be313e5696f000604ec4ccdceac1a24deb1ab  
PCR 11  SHA256  d0e950158f2d66f0c50fce8f91584b2846f852bf3bac9bfbfcfa3750b62b60d0  
PCR 12  SHA256  47e79fa577684c1b35a6c590d5ad65cb5c3bf5ec10535a41fdefcfdb4b44923f  
PCR 13  SHA256  2eb40b3600dba0adb712e85318944b7e205fd00ba72051b5f6a418115ff2cc13  
PCR 14  SHA256  22eeb08994e16121e3bd9a23099080899f4d8cc72159a4792e53e0e9b6e088f0  
PCR 15  SHA256  0000000000000000000000000000000000000000000000000000000000000000  
PCR 16  SHA256  0000000000000000000000000000000000000000000000000000000000000000  
PCR 17  SHA256  ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff  
PCR 18  SHA256  ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff  
PCR 19  SHA256  ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff  
PCR 20  SHA256  ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff  
PCR 21  SHA256  ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff  
PCR 22  SHA256  ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff  
PCR 23  SHA256  0000000000000000000000000000000000000000000000000000000000000000  
```

PCR events:
```console
% govc host.tpm.report -e -host $host
13  Version             SHA256  570063d73d5dcc0aa9cbb224ae876b3ff0da56617b17a1a64d2180f91210bb80  5451414141414541414141414141414141414141414341414341416f414177414e4141544145634142674232625774696232393041465a4e643246795a53424655316870414467754d43347a4c5441754d4334794d7a6b7a4d7a45794e7742694c6d49774d41413d
12  SoftwareComponent   SHA256  40c25fe95666e2a8aa5d2e6d4d25e7266789939322bc141688d94b3972880fce  b.b00
14  Option              SHA256  6e340b9cffb37a989ca544e6bb780a2c78901d3fb33738768511a30617afa01d  jumpstrt.gz
14  Option              SHA256  01ba4719c80b6fe911b091a7c05124b64eeece964e09c058ef8f9805daca546b  useropts.gz
14  Option              SHA256  6e340b9cffb37a989ca544e6bb780a2c78901d3fb33738768511a30617afa01d  features.gz
13  Version             SHA256  636c50a387391c12989f61cfea38928f36bd4298854d56a4ffbc1ffc0ecb82c2  54674141414145414141414141414141414141414143414143514170414177414e51415441456741426742575457746c636d356c6241425754586468636d556752564e59615141344c6a41754d7930774c6a41754d6a4d354d7a4d784d6a6341617935694d444141
12  SoftwareComponent   SHA256  092e54cf6e413750e3b71be4c7acec3427247b0e3ea2dd0fab255922a44588b4  k.b00
13  SoftwareComponent   SHA256  dc4dcaee5c0cf57322e2d8192b1bd224414cc532f0c80fc02288cd126bdae2c4  uc_intel.b00
13  SoftwareComponent   SHA256  e08b07bd7a5742ad03171934c857a14b2f7b07f893b4fbf9ddb131035a1c5862  uc_amd.b00
13  SoftwareComponent   SHA256  e4e161fac0b65d9546ea048bd4231c878bd10ce1932b4fb0663e9e9cc4444a2a  uc_hygon.b00
13  SoftwareComponent   SHA256  45f95be678f141d190df2d1122280668807cc0442a2ea14eecd41660775fa9bf  procfs.b00
13  SoftwareComponent   SHA256  3417799118c4c4c5bf47eb4c8f470e65e749e6c2579ed5d259f14655b674ce2e  vmx.v00
13  SoftwareComponent   SHA256  80a5dbe43369f1bfaff1772a5fedc799ec718df88738ab8e1afc5fa8c569b20e  vim.v00
13  SoftwareComponent   SHA256  d0ff505c916d8e8a45649a583c1c6e01bd500b76f06d4cca07f2fa2fd0023ff0  tpm.v00
12  SoftwareComponent   SHA256  c43d11f825bf57b930736f9dd23392d91369ecc364973f938089aea616a69837  sb.v00
12  SoftwareComponent   SHA256  876c7c010da7192aaa022f6f17841ffdccb103633e1d7f209a8bf16c1c4f229e  s.v00
13  SoftwareComponent   SHA256  3a2b5c35dfc4f4b825e9f3c5c2d031446a4375c3affb6b0a59d9d8f6fdac6321  atlantic.v00
13  SoftwareComponent   SHA256  8dec30faa4bf325bc8307186fe5c2816103214029d611cd4a841b61e1f59b7e4  bcm_mpi3.v00
13  SoftwareComponent   SHA256  d56495f5d98a574965185fda00d9701df814aee61a5c299d78697b1347fee8d1  bnxtnet.v00
13  SoftwareComponent   SHA256  42340a0970a6b9bd815439f87b9c139f33e28594e2165d254bdd00cfc29e9b52  bnxtroce.v00
13  SoftwareComponent   SHA256  85cb6cff577a5a982c8b075619586c604e5fd9a7d99860e009d59a482fa0b4e5  brcmfcoe.v00
13  SoftwareComponent   SHA256  ea00d48e8819b72509577bad309577649239d9789abc7f5178ec878f1a3e4424  cndi_igc.v00
13  SoftwareComponent   SHA256  2b3e064485899d5aa907a4b3bf5ba1b10f599bc6360101e550da922df20efb90  dwi2c.v00
13  SoftwareComponent   SHA256  a57f186ddc2e30ec794a4019daac2fd7f998e1c13a60fac0e026e3f62871952b  elxiscsi.v00
13  SoftwareComponent   SHA256  28a90f615a595f85c37752a91e94ed3801d7e9ad3ded779b2917211c0a5a3738  elxnet.v00
13  SoftwareComponent   SHA256  1afc9d0880406a21057d8fa0ac7e19b8ee4d3a38fa9be2b446db14189e495916  i40en.v00
13  SoftwareComponent   SHA256  7f8fdc32b43e9eb172259a76bf0a32637c8120e13713f9eb46678786a43f00c2  iavmd.v00
13  SoftwareComponent   SHA256  4fa712d553fad31399daea39b1ec21b131b1275a7b28d56757d31e3dece3d444  icen.v00
13  SoftwareComponent   SHA256  1624155a2656f9107d0e60a629c2a510f3c4a5bb139ab4cdc661bd38a624c0ce  igbn.v00
13  SoftwareComponent   SHA256  7af0fce53ce1a7e520192cf51935da28df4c7bc6c213a2cbd27ffe5d377da95c  intelgpi.v00
13  SoftwareComponent   SHA256  4a2758944f3eb39db5377aa573491b956664a471c8948c1ad751db97e7f7f167  ionic_cl.v00
13  SoftwareComponent   SHA256  68bf0f1a21e3ee477ca817247ee9af1cd1fe0c51dc27c014d1e514ee7316dfa7  ionic_en.v00
13  SoftwareComponent   SHA256  3ea3416b803f63183f957eb79c37b569b520697aa73c4bba7eba51c48620d776  irdman.v00
13  SoftwareComponent   SHA256  a8f6bcee4851ed79a203e262bc6f73cdb4af0c38f752adcd7b8c26fba82ca6bc  iser.v00
13  SoftwareComponent   SHA256  f4dcfaa49cf44e67c3ed6f97d0cb88a46b4376cf482c12d08a252894377f27cc  ixgben.v00
13  SoftwareComponent   SHA256  b6e2f7c401298ea53494424d40efa93eed2cc344a556f30747c1fa5ec8444190  lpfc.v00
13  SoftwareComponent   SHA256  cd6bca72898943c18b319797e920bb96756e584973f4971e92f546cbc5c347e0  lpnic.v00
13  SoftwareComponent   SHA256  38dff31308274d1b08ae029a202f7c6e8234cde1f0880eace7a678c693c3f1c5  lsi_mr3.v00
13  SoftwareComponent   SHA256  d56cd47f8cd9b25d5f51dcb765fa3f57ea85f862fa3fe161a828390211ad002f  lsi_msgp.v00
13  SoftwareComponent   SHA256  d72d09fec2c01ac23bb9dbd22c1839298171aeb221cf1b2f2a0d296a23636885  lsi_msgp.v01
13  SoftwareComponent   SHA256  c68d9a1e58e941f2825a25a999bea802447bc9a621cca57e04ad75e0eada3491  lsi_msgp.v02
13  SoftwareComponent   SHA256  2f9d13684a2479f1795bd8492edcbdbe2f706e2bd037f60e9c12a7b1e42cb80a  mtip32xx.v00
13  SoftwareComponent   SHA256  7beaeb9931189f42d0d8cd6341ea8ea1832bb2d26160ba8ab6c0b6ec16f612fa  ne1000.v00
13  SoftwareComponent   SHA256  abaa7763686036fdd715c20c51c76b83d953ee8a15375353ead48f7a74764d5c  nenic.v00
13  SoftwareComponent   SHA256  b48fe38905f920fb17f036203c878d9ce6412c4475f323f3861ed9c32037f6ec  nfnic.v00
13  SoftwareComponent   SHA256  805a4639ad40440bdc3bc5ce918820357325685c05abc1bd99b0460ab062c7ef  nhpsa.v00
13  SoftwareComponent   SHA256  d94352c4fac8c86841d6f359ad5fb90d66fc6eef6c35edc6e02eb7b48ea5d479  nipmi.v00
13  SoftwareComponent   SHA256  dabbd7db392a3f3d493ed06b45e1577ad09d20683f7f8edd33e29d957fafda0b  nmlx5_cc.v00
13  SoftwareComponent   SHA256  e3ff3663e76b146b337adf600f187117880f05b0b1b507e52d6fbfc0faf3fcf9  nmlx5_co.v00
13  SoftwareComponent   SHA256  05eb3c65f3957567f65b109b02d4fbca8a58fc07b7b6975705dc6ad94609c589  nmlx5_rd.v00
13  SoftwareComponent   SHA256  cb5784465884c80c70973340724a20920189e303aa2f5c6dff002613903eaef3  ntg3.v00
13  SoftwareComponent   SHA256  981b60a4de88b68b0f56c7dddd41cef9cba0b38a8c3f1ed3a915e783e15618f9  nvme_pci.v00
13  SoftwareComponent   SHA256  6b452b09c326bdf4313fd13505b71d8c0ca3da64039be50da81aa4e4974ddfe2  nvmerdma.v00
13  SoftwareComponent   SHA256  c4718c87146db73001eabb89988bd99c3c6e640099ac7e62070af07d91ffec11  nvmetcp.v00
13  SoftwareComponent   SHA256  772749ccf09ba64cd6da7674096ca8072af568a124913f416bf9aa766feeba16  nvmxnet3.v00
13  SoftwareComponent   SHA256  a9236799b8be5e9f6010894529de8bb9c5149e954c681aba2f592d021a6d7969  nvmxnet3.v01
13  SoftwareComponent   SHA256  cc6a7be8e387bacff6039891be0543765e6160f222cd74172e8e82d6524a2777  pvscsi.v00
13  SoftwareComponent   SHA256  26da18540f7e55e38cb1b6c78c283bf71f03e1bc8ec34380d6a957cac0713fc4  qcnic.v00
13  SoftwareComponent   SHA256  cef0c4bf5f0bb34bb68d8e3baf5b6ba3df055a29e7bc1bc450e2c2aa24766a15  qedentv.v00
13  SoftwareComponent   SHA256  151176b65c4da764e196ab21bfeeaeef32b38baeb1a93377daeb75b3bcf287b6  qedrntv.v00
13  SoftwareComponent   SHA256  18af667a0bba559d99a8e0e99f17ce387c4016739d12c7ee2f89aa46de084663  qfle3.v00
13  SoftwareComponent   SHA256  cac7dfa039a50f1676e179438e9f1d092fcb461b03fe1fdd7c0c4ccf3fc47103  qfle3f.v00
13  SoftwareComponent   SHA256  c60e17706d872d08bc4128a140ada24b7558bf2e4aad83986487dc4f7bc8d02f  qfle3i.v00
13  SoftwareComponent   SHA256  4a9cb08b9d1e190d296e44eb20ada8d2152c9d2592589154ae905ace84c841c9  qflge.v00
13  SoftwareComponent   SHA256  3d4d864a3d39d922e282ece5d53b0711cce0cb93fac7e583ea85586504ae3f43  rdmahl.v00
13  SoftwareComponent   SHA256  2caadf1412a9d3e4a13621f560de1fb7ad73a1d6c79feb9183a5171e191a7053  rste.v00
13  SoftwareComponent   SHA256  7532641ea62efb7d7cec7a0c7b5c817298556a4ca1e7f37a59f7db0d4a07b829  sfvmk.v00
13  SoftwareComponent   SHA256  8d2ba169bcb66f07c97614f96464a8023e41c872c44449cad13a704ae150095e  smartpqi.v00
13  SoftwareComponent   SHA256  a6a0b99a20f89739b14bed809c90810c225bda02a5e3cdb630835d2f925529ae  vmkata.v00
13  SoftwareComponent   SHA256  5504e6665507cd241589f99d3a6dd0228b80be1d934e6a9c7aa9bf831334a85c  vmksdhci.v00
13  SoftwareComponent   SHA256  bd058bfe5efa2a7a0886217ec8960c39097e85e3d563d8651121f81d323fd2b3  vmkusb.v00
13  SoftwareComponent   SHA256  12292858d088bd4e8a21396e4009f80ad76a3b2e6616da3672ed0810561efbfe  vmw_ahci.v00
13  SoftwareComponent   SHA256  eb9122a118496b15681776b11497b38a8c5ee92262c925b348d4e52685bc9744  bmcal.v00
13  SoftwareComponent   SHA256  e7e5cbd6802107dac7d4330536ee17dbe278282ad859893f3f6168f2b6df67db  clusters.v00
13  SoftwareComponent   SHA256  2e9159037a37c99c8caa2368e5171db5589444c0f7b26daa0b5b568a901b753f  crx.v00
13  SoftwareComponent   SHA256  599862a7f9d65e55c94382a4b6315c1fda40104bd090ce86b87bcf885f9c8823  drivervm.v00
13  SoftwareComponent   SHA256  f73ba9704e8103f85716795959434a91b7b54343176854a2fb5eab95beedd2ca  elx_esx_.v00
13  SoftwareComponent   SHA256  1d7bdc9243308c46dfba04222fee4c1a01f98c742bf17bfaf8f45e1a201ee662  btldr.v00
13  SoftwareComponent   SHA256  7ed454bf0d7b6cf5db3fc56c9fd04e5e634e228f3fd74b41eb59c32e1f4edb3b  dvfilter.v00
13  SoftwareComponent   SHA256  287276ebeb7dc93d2438ab9c881e6fb16349d1a2ec74db207d26023e38988f62  esx_ui.v00
11  SoftwareComponent   SHA256  b2573868e7b49ea39743c28f7112e4469772e016c4b6676b102a8b3ffef686ec  esxupdt.v00
13  SoftwareComponent   SHA256  753d4b9c822419f7f5271cd76a8124a2a01c45e2cad68465a9910b6f3006fbe5  tpmesxup.v00
13  SoftwareComponent   SHA256  acf1780f11462b5438919d39403c0d137338b398ec64dd91403073ab17e29a2f  weaselin.v00
13  SoftwareComponent   SHA256  b9a1d189d4c03fc5751545fdca10537c2724ab641d28c4573ec3dc9d83f5a467  esxio_co.v00
13  SoftwareComponent   SHA256  5b053500f50eeadf3375d069a89c0f12a19df3646b3c2e329214435fc4c74334  infravis.v00
13  SoftwareComponent   SHA256  5bab1ec02b97a90d0818f36ac5738ca8acefcf3e9a2bbb4ccfd8956cc911871c  loadesx.v00
13  SoftwareComponent   SHA256  a9730871ee21006b1d89250e653586fecf80cc054a4f237b510cff839e40e03c  lsuv2_hp.v00
13  SoftwareComponent   SHA256  b6801f095590d6ea4c9298ef771a7e46e77896b9c0ec3c8dd8f17de2471e4150  lsuv2_in.v00
13  SoftwareComponent   SHA256  16fe1218c2ee43fab6092c51d6d32e080aa9d51fd06826688ca8f5f6fb2d8cae  lsuv2_ls.v00
13  SoftwareComponent   SHA256  e446f11ab99bb642f61c34eb04608b63bced93d1ba0ad7599c76e9d74a69108b  lsuv2_nv.v00
13  SoftwareComponent   SHA256  56de0d3ee3c99ca3d37b6abe8a88ed0db2d114d974a1f4f0ca849f7551a51838  lsuv2_oe.v00
13  SoftwareComponent   SHA256  e5c3d3619e0e0b7ab1327551523d1d08f29f3b31ed2809d151624cc5d01b875b  lsuv2_oe.v01
13  SoftwareComponent   SHA256  257b53059986b9b1f773769ac988f29591904e676fb6cd51c8efd1476984aaa1  lsuv2_sm.v00
13  SoftwareComponent   SHA256  e01caad7da0884b8fbd3ed36fa5592a7cc587393485504d8cae8aa035a4642b0  native_m.v00
13  SoftwareComponent   SHA256  0c0fc442f6d43bbca2dbcb9a2b5e11edc7bab5ff8b6940429da38558f352494d  qlnative.v00
13  SoftwareComponent   SHA256  2ac2fcc226e1f3fee69e202d7aafad1812d7515e06cfdd9ed7c8dd0dcf9842e5  trx.v00
13  SoftwareComponent   SHA256  19b4415ec5e76b9234b395572d057ac3021c7d7c38081cb4642c2a6782459c52  vcls_pod.v00
13  SoftwareComponent   SHA256  334e0d1af1072e6d7d315f878700f67c5012c514635c9f9863438217d3673c87  vdfs.v00
13  SoftwareComponent   SHA256  175c224ed50d1775a9136ae4732d68e347c148a308c85200f2309a3c19cc499a  vds_vsip.v00
13  SoftwareComponent   SHA256  2e15e8cc5585c04317ed4b582b7e9d8af659460d4fb41e3db950edb19188b55d  vmware_e.v00
13  SoftwareComponent   SHA256  ce5f337bf6c2256fc901f7c5a1bcf735ad646490658e2e79a423685b50a5fe59  hbrsrv.v00
13  SoftwareComponent   SHA256  18d42dbeed68ba06a58459f926b852ad9e520563c208cbc7ac7df4a590d915c6  vsan.v00
13  SoftwareComponent   SHA256  4165580f7e5431e50cb7bde160586fe5fe19cee26f86a75d93e88fd2993d6939  vsanheal.v00
13  SoftwareComponent   SHA256  8f38e2b0905122ebc73d154dfc17abdcd600f475122b86f1c7ca5f305b5a4e07  vsanmgmt.v00
13  SoftwareComponent   SHA256  096b97ada06fd8daabbea2458f8da64f1bdf64109f11ee19519909bc16d5c542  xorg.v00
13  SoftwareComponent   SHA256  5f353f015a4186db871d3106e6d17d84f71b21d6e6f635786f660daa25c058c2  gc.v00
13  SoftwareComponent   SHA256  2e480abda0b193662c3fd724a655bb1af8e8e06ca90a63a9e5d5cc211912b598  imgdb.tgz
13  SoftwareComponent   SHA256  5870236cf95aea775ad7b4c97f1191f5d3f085d59365865573b080075e9ecb9f  basemisc.tgz
14  Signer              SHA256  321690a8e06810b184032f073c305a082112bdad5870ef2e8953e01be956d710  signer = TRUE
14  Command             SHA256  985a8acc2fff75dad8e0016e15285ecde7757d2c229121ffdcdde56eba69b4e7  
14  BootSecurityOption  SHA256  f0e0032641b9438941346480ae1564eee57ab378abb155c7cd49dbdd770d55af  execInstalledOnly = FALSE
10  BootSecurityOption  SHA256  f0e0032641b9438941346480ae1564eee57ab378abb155c7cd49dbdd770d55af  execInstalledOnly = FALSE
14  BootSecurityOption  SHA256  f0ae8cb4bb9a16ddc90443ccad01fbe5739250e5eaae1d6f06f4f6bb7af10acb  uefiSecureBootEnabled = TRUE
10  BootSecurityOption  SHA256  f0ae8cb4bb9a16ddc90443ccad01fbe5739250e5eaae1d6f06f4f6bb7af10acb  uefiSecureBootEnabled = TRUE
14  BootComplete        SHA256  3c3c02fc92bb82d126b7333dddd788a3a64d27b01ac7e46787a5176e5b401281  

```
